### PR TITLE
Update teams.yaml

### PR DESCRIPTION
**Pull request by Nivedita-coder**
Updated the `ci-signal` as `release-team-ci-signal`

/cc @leonardpahlke @reylejano

### CODE REVIEW

**Comment by leonardpahlke** [comment]
Thanks @Nivedita-coder :)

/hold 

I think some more references needs to be updated in other repos (ref @reylejano comment https://kubernetes.slack.com/archives/C2C40FMNF/p1664383282833649?thread_ts=1664382614.855319&channel=C2C40FMNF&message_ts=1664383282.833649)

**Comment by palnabarun** [comment]
do we have any other references to the team?

**Comment by leonardpahlke** [comment]
not that I know of

**Comment by mrbobbytables** [comment]
is this still a thing? the PR is pretty old at this point

**Comment by leonardpahlke** [comment]
Yes. I think, we should merge this so that the sig-release/release-team teams are named consistently.

**Comment by leonardpahlke** [comment]
FYI 1.27 ci-signal lead @lauralorenz

**Comment by reylejano** [comment]
> is this still a thing? the PR is pretty old at this point

@mrbobbytables I think this PR is still valid. This PR reflects a name change for ci-signal to add the `release` prefix so that it aligns with the other `release` related teams/subteams
https://github.com/kubernetes/org/blob/main/config/kubernetes/sig-release/teams.yaml#L301-L335

/cc @salaxander @JamesLaverack

File: config/kubernetes/sig-release/teams.yaml [hunk 1]
```diff
@@ -336,7 +336,7 @@ teams:
         - xmudrii # Release Manager
         privacy: closed
         teams:
-          ci-signal:
+          release-team-ci-signal:
```
Comment (mrbobbytables) [inline config/kubernetes/sig-release/teams.yaml:339]: the name of the team can't be changed directly like this. It needs to add a previously

https://github.com/kubernetes/org/blob/f8b8c12cbcd745d0fbbf0b219b78b13f3ab628db/config/kubernetes/sig-architecture/teams.yaml#L70-L71

Also please make the commit message a bit more clear than 'update the name'.

**Comment by nikhita** [comment]
@Nivedita-coder can you update the PR based on the changes requested?

**Pull request closed without merging**